contribs/gmf/src/d3-lidar-profile/lidarprofileMeasure.js

```diff
@@ -0,0 +1,200 @@
+goog.provide('gmf.lidarProfile.measure');
+
+
+/**
+* Measure tool for the d3 chart
+* @constructor
+* @param {Object} options to be defined in gmfx
```

Comment:
to be defined

Reply:
Idem, help for gmfx type definition needed